tgui/packages/tgui/interfaces/BankingComputer/type.ts

```diff
@@ -69,4 +69,5 @@ export interface BankComputerStatus {
   authenticated: boolean,
   cardInserted: boolean,
   cardName: string,
+  isSiliconUser: boolean,
```

Comment:
Noting here, though this applies to a bunch of this file: data that comes from `ui_data` is given with `BooleanLike` instead of `boolean`.